libs/akita-ng-effects/src/__tests__/actions.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { Actions } from '@datorama/akita-ng-effects';
+import { fakeAsync, flush } from '@angular/core/testing';
+
+describe('Actions', () => {
+  it('should notify on dispatched event', fakeAsync(() => {
+    const actions = new Actions();
```

Comment:
Why we need `fakeAsync` here? It's sync operation.

Reply:
As far as I'm concerned this is async.
The subject publishes the event and the assertion should run inside the async callback of the subscription.
It results in a false positive once the fakeAsync is removed.
The assertion won't run.